learn/app-development/widgets/container/tabs.md

```diff
@@ -2,136 +2,153 @@
 title: "Tabs"
 id: ""
 ---
+---
+
+**Tabs** can be used when you need multiple containers within a single window. Tab-based navigation provides an easy and powerful mechanism to handle a large amount of content within a limited area by separating content into different panes where one pane is viewable at a time. The user can quickly access the content by switching between the panes without leaving the page.
+
+In interface design, a Tabbed Document Interface (TDI) or Tab is a graphical control element that allows multiple documents or panels to be contained within a single window,  using Tabs as a navigational widget for switching between sets of documents. It is an interface style most commonly associated with a web browser, web application and text-editor, and preference panes.
 
-**Tabs** can be used when you need multiple containers within a single window. Tab-based navigation provides an easy and powerful mechanism to handle a huge amount of content within a limited area through separating content into different panes where one pane is viewable at a time. The user can quickly access the content by switching between the panes without leaving the page.
+[![tab run](/learn/assets/tab_run.png)](/learn/assets/tab_run.png)
 
-In interface design, a tabbed document interface (TDI) or Tab is a graphical control element that allows multiple documents or panels to be contained within a single window,  using tabs as a navigational widget for switching between sets of documents. It is an interface style most commonly associated with a web browser, web application and text-editor, and preference panes.
+### Dynamic Tabs
 
-[![](/learn/assets/tab_run.png)](/learn/assets/tab_run.png)
```

Comment:
Before directly introducing dynamic tabs, it would be better to mention what are static and dynamic tabs and how are they different, along with use cases depicted in a picture.

Reply:
Added that in the introduction part as [Types of Tabs](https://pr-469.d2ulx3keu6fdjg.amplifyapp.com/learn/app-development/widgets/container/tabs#types-of-tabs), which explains how they are different. 

Separate use cases is a long story, but the static use case covers in the same document. We must only verify if it is correct.